errors/errors.go

```diff
@@ -541,3 +541,9 @@ var ErrNilPersistentHandler = errors.New("nil persistent handler")
 
 // ErrNilGenesisNodesSetupHandler signals that a nil genesis nodes setup handler has been provided
 var ErrNilGenesisNodesSetupHandler = errors.New("nil genesis nodes setup handler")
+
+// ErrNilLogger signals that a nil logger instance has been provided
+var ErrNilLogger = errors.New("nil logger")
+
+// ErrNilShuffleOutCloser signals that a nil shuffler out closer has been provided
+var ErrNilShuffleOutCloser = errors.New("nil shuffler out closer")
```

Comment:
should we go with `shuffler out` or with `shuffle out` ?

Reply:
updated, typo